modules/service/src/main/scala/lucuma/odb/service/GuideEnvironmentService.scala

```diff
@@ -317,6 +319,7 @@ object GuideEnvironmentService {
       ): F[Either[Error, GuideEnvironment]] =
         (for {
           obsInfo        <- EitherT(getObservationInfo(pid, oid))
+          site           <- EitherT.fromEither(obsInfo.site)
```

Comment:
This is what introduces the "No configuration defined ..." message?

Reply:
Yes. I get the site by knowing what the configuration is. Is there a better way of knowing the site that I should use?

Reply:
No, I was just looking to find how the message would be triggered but I see it's via the `site` which asks for the `fpu`.  You're using the `fpu` anyway to define the AGS params.

For what it's worth, the observation also directly defines the `c_observing_mode_type` which will tell you `GmosNorthLongSlit` or `GmosSouthLongSlit` or neither.  The observing mode defines which instrument it applies to, and we probably need something that facilitates mapping from `Instrument` to `Site` somewhere at some point if it doesn't exist.

Reply:
Let me see if there is a mapping from Instrument to Site and I'll add that into my next PR if possible. I guess `c_observing_mode_type` would be null if there isn't a configuration?

Reply:
OK. I couldn't find one either.